lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,14 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cisco5-64' => {
+          'platform' => 'cisco-5-x86_64',
+          'template' => 'cisco-nxos-9k-x86_64'
+        },
+        'cisco7-64' => {
```

Comment:
@joshcooper I agree that we may need to be more specific in the platform string, but if that's the case then we will also have to update beaker here:
https://github.com/puppetlabs/beaker/blob/54b1801d9bb6187796142160d3c0828639c542d5/lib/beaker/platform.rb#L6